docs/en/siem/field-ref.asciidoc

```diff
@@ -1,23 +1,24 @@
 [[siem-field-reference]]
 [chapter, role="xpack"]
-= Field reference and mappings
+= SIEM field reference guide
 
-This section lists all fields required to populate {siem-soln} data.
+This section lists ECS fields the {siem-app} uses to display data.
 
-NOTE: Unless noted otherwise, all listed fields are {ecs-ref}[ECS fields]. 
+IMPORTANT: It is recommended to use {beats} to ship your data. When you use
+{beats}, manual field mapping is not required and all {siem-app} data is
```

Comment:
I'd make it clearer that if you are using any of our out-of-the-box [Beats modules](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-modules.html), the data is already in ECS. If you are ingesting data from a custom data source, mapping to ECS is still required. Could link to [ECS Mapper](https://github.com/elastic/ecs-mapper) which simplifies the mapping process.